src/fault_injector/fault_types/fault.py

```diff
@@ -37,3 +40,21 @@ def next_tick(self, tick: int):
             self.inject_fault(self.component)
         elif tick == self.end_tick:
             self.resolve_fault(self.component)
+
+
+class FaultConfiguration(BaseModel):
```

Comment:
Please add the FaultConfiguration class to the array of db-tables in `src/constants.py` 😇

Reply:
Not needed, because FaultConfiguration should be abstract and not available as table.